### PR TITLE
add mappings for model

### PR DESCRIPTION
**Pull request by miltontony**

### CODE REVIEW

**Comment by miltontony** [comment]
@smn review please

File: mappings/category.mapping.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    "language": {
+      "index": "not_analyzed",
+      "type": "string"
+    },
+    "uuid": {
+      "index": "not_analyzed",
+      "type": "string"
+    }
+  }
+}
```
Comment (smn) [inline mappings/category.mapping.json]: missing newline nitpick.

File: mappings/page.mapping.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    "id": {
+      "index": "not_analyzed",
+      "type": "string"
+    },
+    "primary_category": {
+      "index": "not_analyzed",
+      "type": "string"
+    }
+  }
+}
```
Comment (smn) [inline mappings/page.mapping.json]: missing newline nitpick.

**Comment by smn** [comment]
:+1:

**Pull request merged**